docs/conf/portland/2023/news/welcome-get-ready.rst

```diff
@@ -0,0 +1,79 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/venue.png
+
+.. post:: May 1, 2023
+   :tags: {{shortcode}}-{{year}}
+
+Welcome to Write the Docs Portland 2023
+=======================================
+
+We're 1 week out from the conference, and we hope everyone is getting excited!
+
+In this update we discuss a change to the hike, share details on the livestream, and a few other updates as the conference approaches.
+
+Hike Reroute
+------------
+
+Unfortunately, there is a closure of the trail that we were planning to hike because of a large rain event this spring.
+This means the hike will turn into an "out and back" hike from Lower Macleay Trailhead to Pittock Mansion.
+The options for public transit at the bottom of the trail are a bit limited,
+so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
+
```

Comment:
You reverted my suggested change to this, intnentionally?

Reply:
Bleh, no. No sure how they got lost :(

Reply:
Re-added.